synapse/tests/test_tools_genpkg.py

```diff
@@ -78,6 +78,11 @@ async def test_tools_genpkg(self):
             msgs = await core.stormlist('$mod=$lib.import(testmod) $lib.print($mod)')
             self.stormIsInPrint('Imported Module testmod', msgs)
 
+            gdefs = await core.callStorm('return($lib.graph.list())')
```

Comment:
is this really testing the genpkg tool?

Reply:
it is testing that the graph definition was processed and loaded correctly, similar to the command/module tests above it. I can move the delete test below though